configure.ac

```diff
@@ -864,12 +864,12 @@ CPPFLAGS="\
 
 AC_SUBST([OPTIONAL_MATH_CFLAGS])
 AC_SUBST([OPTIONAL_MATH_LIBS])
-AC_SUBST([OPTIONAL_RT_CLFAGS])
+AC_SUBST([OPTIONAL_RT_CFLAGS])
```

Comment:
This looks dead option, if you agree we should take the opportunity to remove it

Reply:
Yeap, it's dead.